packages/neuron-ui/src/components/History/RowExtend.tsx

```diff
@@ -75,93 +81,98 @@ const RowExtend = ({ column, columns, isMainnet, id, bestBlockNumber }: RowExten
   }, [hash, dispatch])
 
   useEffect(() => {
-    if (status !== 'success') {
-      if (column.type === 'send' && !column.nftInfo && !column.nervosDao) {
-        getOnChainTransaction(hash).then(tx => {
-          // @ts-expect-error Replace-By-Fee (RBF)
-          const { min_replace_fee: minReplaceFee } = tx
-          if (minReplaceFee) {
-            setAmendabled(true)
-          }
-        })
-      }
+    if (status !== 'success' && column.type !== 'receive' && !isWatchOnly) {
+      getOnChainTransaction(hash).then(tx => {
+        // @ts-expect-error Replace-By-Fee (RBF)
+        const { min_replace_fee: minReplaceFee } = tx
+        if (minReplaceFee) {
+          setAmendabled(true)
+        }
+      })
     }
     setAmendabled(false)
```

Comment:
This line `setAmendabled(false)` will execute every time. It will be better to move this line to the top of `useEffect`, then it will be better clear.